configuration/sitemaps.md

```diff
@@ -506,29 +529,33 @@ visibility=[Day_Time=="Morning", Day_Time=="Afternoon", Temperature>19]
 ```
 
 If any one of the comparisons is evaluated as `true`, the Item will be visible, otherwise it will be hidden.
-It is important to note, that it is not possible to decide visibility on more than one condition at the same time.
-The third example might be the visibility of a sprinkler control.
-The control will be visible if it is Morning *OR* if it is Afternoon *OR* if the temperature is above 19°C.
-To achieve more complex conditions, you will benefit from defining a helper Item and a rule to set it.
+It is important to note that visibility may only be determined based on one condition at a time.
 
-Valid comparison operators are:
+The third example might control the visibility of a lawn sprinkler control.
+The control will be visible if it is Morning *OR* if it is Afternoon *OR* if the temperature is above 19°C.
+To achieve more complex conditions, you will benefit from defining a helper Item and a [rule]({{base}}/configuration/rules-dsl.md) to set it.
 
-- equal to `==`, unequal to `!=`
-- smaller or equal to `<=`, bigger or equal to`>=`.
-- smaller than `<`, bigger than `>`
+<!-- TODO: Provide an example for the statement above or remove it.  It is not that helpful as it is now for people who are just getting aquainted with sitemaps -->
 
 ### Label and Value Colors
 
 Colors can be used to emphasize an items label or its value based on conditions.
+Colors may be assigned to either the label or the value associated with an Item.
+
+Label and Value Color Syntax:
```

Comment:
See above

Reply:
I do not understand this comment.

Reply:
Bold?

Reply:
Understood - this comment has been addressed.